hooks/playbooks/adoption_ironic_post_oc.yml

```diff
@@ -0,0 +1,124 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: OSP 17 - Ironic post overcloud
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  tasks:
+    - name: Gather ansible_user_dir from undercloud
+      delegate_to: osp-undercloud-0
+      ansible.builtin.setup:
+        gather_subset:
+          - user_dir
+
+    - name: Glance images - ironic-python-agent
+      delegate_to: osp-undercloud-0
+      block:
+        - name: Install the ironic-python-agent images package
+          become: true
+          ansible.builtin.dnf:
+            name: rhosp-director-images-ipa-x86_64
+            state: present
+
+        - name: Create ironic-python-agnet directory
+          ansible.builtin.file:
+            state: directory
+            path: "{{ ansible_user_dir }}/ironic-python-agent"
+
+        - name: Extract ironic-python-images from tar
+          ansible.builtin.unarchive:
+            src: /usr/share/rhosp-director-images/ironic-python-agent-latest.tar
+            dest: "{{ ansible_user_dir }}/ironic-python-agent"
+            remote_src: true
+
+        - name: Upload ironic-python-agent kernel to glance
+          environment:
+            OS_CLOUD: overcloud
+          ansible.builtin.shell:
+            cmd: >-
+              openstack image show deploy-kernel &>/dev/null || \
+                openstack image create deploy-kernel \
+                  --public \
+                  --container-format aki \
+                  --disk-format aki \
+                  --file ironic-python-agent.kernel
+          args:
+            chdir: "{{ ansible_user_dir }}/ironic-python-agent"
+
+        - name: Upload ironic-python-agent ramdisk to glance
+          environment:
+            OS_CLOUD: overcloud
+          ansible.builtin.shell:
+            cmd: >-
+              openstack image show deploy-ramdisk &>/dev/null || \
+                openstack image create deploy-ramdisk \
+                  --public \
+                  --container-format ari \
+                  --disk-format ari \
+                  --file ironic-python-agent.initramfs
+          args:
+            chdir: "{{ ansible_user_dir }}/ironic-python-agent"
+
+    - name: Create ironic provisioning network
+      delegate_to: osp-undercloud-0
+      environment:
+        OS_CLOUD: overcloud
```

Comment:
You have parameter for OS_CLOUD name defined in https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/scenarios/uni01alpha.yaml#L37 
`overcloud` should be replaced with `cifmw_adoption_osp_deploy_scenario.stacks[0].stackname`

Reply:
Done.